backend/handler/share/chat.go

```diff
@@ -91,14 +90,9 @@ func (h *ShareChatHandler) ChatMessage(c echo.Context) error {
 		return h.sendErrMsg(c, "invalid app type")
 	}
 	ctx := c.Request().Context()
-	appInfo, err := h.appUsecase.GetAppDetailByKBIDAndAppType(ctx, req.KBID, domain.AppTypeWeb)
-	if err != nil {
-		return h.sendErrMsg(c, "failed to get app info")
-	}
-	if appInfo.Settings.CaptchaSettings.ChatStatus == consts.CaptchaStatusEnable {
-		if !h.Captcha.ValidateToken(ctx, req.CaptchaToken) {
-			return h.sendErrMsg(c, "failed to validate captcha")
-		}
+	// validate captcha token
+	if !h.Captcha.ValidateToken(ctx, req.CaptchaToken) {
```

Comment:
在移除基于应用设置的验证码验证逻辑后，所有聊天和评论请求都会无条件验证验证码。这可能会导致在不需要验证码的场景下出现不必要的验证步骤，建议恢复基于设置的条件验证逻辑，或提供其他机制来控制验证码的使用。

```suggestion
// validate captcha token if captcha is enabled
if shouldValidateCaptcha(appInfo) && !h.Captcha.ValidateToken(ctx, req.CaptchaToken) {
    return h.sendErrMsg(c, "failed to validate captcha")
}